runtime/startup_byt.c

```diff
@@ -376,6 +376,15 @@ static int parse_command_line(char_os **argv)
   return i;
 }
 
+static const char_os * get_stdlib_location(void)
+{
+  const char_os * stdlib;
+  stdlib = caml_secure_getenv(T("OCAMLLIB"));
+  if (stdlib == NULL) stdlib = caml_secure_getenv(T("CAMLLIB"));
```

Comment:
For portable consistency, can we check for NULLs and empty strings? (Unless it is documented somewhere that empty OCAMLLIB stops search for CAMLLIB and OCAML_STDLIB_DIR, which would be weird).